tracks/cpp/exercises/atbash-cipher/atbash-cipher/mentoring.md

```diff
@@ -0,0 +1,4 @@
+The atbash cipher is symmetrical that is why it is possible to use one function to encode and decode.
```

Comment:
* This doesn't follow the structure of other mentoring notes. 
* These notes are _very_ sparse. If you'd like to tackle creating new notes, it would be nice to have them a bit more fleshed out.